ddl/table.go

```diff
@@ -321,6 +325,7 @@ func (w *worker) onRecoverTable(d *ddlCtx, t *meta.Meta, job *model.Job) (ver in
 			}
 		})
 
+		job.CtxVars = []interface{}{tids}
```

Comment:
The rule cache should be recovered when recovering the table.

Reply:
Yes, it is at L85~L89, and L161~L165.